src/python/grpcio/grpc/_channel.py

```diff
@@ -302,6 +312,41 @@ def __repr__(self):
     def __str__(self):
         return self._repr()
 
+    def cancel(self):
+        """See grpc.Future.cancel."""
+        return False
+
+    def cancelled(self):
+        """See grpc.Future.cancelled."""
+        return False
```

Comment:
Should this function return the cancellation state of the `RpcError` or the RPC call? Apply to other methods.

Reply:
Discussed offline. This class is supposed to have a very limited scope. The name has been changed to `_InactiveRpcError` to indicate this.